test/services/user/forcePasswordChange.test.js

```diff
@@ -0,0 +1,90 @@
+const { expect } = require('chai');
+const app = require('../../../src/app');
+const forcePasswordChangeService = app.service('forcePasswordChange');
+const accountService = app.service('accounts');
+const testObjects = require('../helpers/testObjects')(app);
+const { generateRequestParamsFromUser } = require('../helpers/services/login')(app);
+
+describe('forceChangePassword service tests', () => {
+	let server;
+	const newPassword = 'SomePassword1!';
+	const newPasswordConfirmation = 'SomePassword1!';
+	const newPasswordThatDoesNotMatch = 'SomePassword2!';
+	const newPasswordThatIsToWeak = 'SomePassword1';
+
+	before((done) => {
+		server = app.listen(0, done);
+	});
+
+	after((done) => {
+		server.close(done);
+	});
+
+	const postChangePassword = (requestParams, password, password2) => forcePasswordChangeService.create({
+		'password-1': password,
+		'password-2': password2
+	}, requestParams, app);
+
+	describe('CREATE', () => {
+		it('when the passwords do not match, the error object is returned with the proper error message', async () => {
+			const testUser = await testObjects.createTestUser();
+			const userRequestAuthentication = await generateRequestParamsFromUser(testUser);
+			return postChangePassword(userRequestAuthentication, newPassword, newPasswordThatDoesNotMatch)
+				.catch((err) => {
+					expect(err.code)
+						.to
+						.equal(400);
+					expect(err.name)
+						.to
+						.equal('BadRequest');
+					expect(err.message)
+						.to
+						.equal('Die neuen Passwörter stimmen nicht überein.');
+				});
+		});
+		it('when the password is to weak, the error object is returned with the proper error message', async () => {
+			const testUser = await testObjects.createTestUser();
+			const userRequestAuthentication = await generateRequestParamsFromUser(testUser);
+			return postChangePassword(userRequestAuthentication, newPasswordThatIsToWeak, newPasswordThatIsToWeak)
+				.catch((err) => {
+					expect(err.code)
+						.to
+						.equal(400);
+					expect(err.name)
+						.to
+						.equal('BadRequest');
+					expect(err.message)
+						.to
+						.equal('Dein Passwort stimmt mit dem Pattern nicht überein.');
+				});
+		});
+		it('when the user has been forced to change his password, the proper flag will be setted after changing the password', async () => {
```

Comment:
Codacy found an issue: [This line has a length of 140. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5748939)